src/Raven.Server/Config/Categories/CoreConfiguration.cs

```diff
@@ -52,6 +52,27 @@ public class CoreConfiguration : ConfigurationCategory
         [ConfigurationEntry("ThrowIfAnyIndexOrTransformerCouldNotBeOpened")]
         public bool ThrowIfAnyIndexOrTransformerCouldNotBeOpened { get; set; }
 
+        [Description("Disable Kestrel's minimum required data rate with grace time enforcement. If set to true this option overrides both MinDataRateBytesPerSecond and MinDataRateGracePeriod settings")]
+        [DefaultValue(false)]
+        [ConfigurationEntry("MinDataRateDisable")]
+        public bool MinDataRateDisable { get; set; }
+
+        [Description("Set Kestrel's minimum required data rate in bytes per second. This option should configured together with MinDataRateGracePeriod. If MinDataRateDisable set to true - this option will be ignored")]
+        [DefaultValue(0)]
+        [ConfigurationEntry("MinDataRateBytesPerSecond")]
+        public int MinDataRateBytesPerSecond { get; set; }
```

Comment:
Let's use `TimeSetting`, then you can avoid `TimeSpan.FromSeconds()`. Also for consistency with other names let's name it `MinDataRateBytesPerSec`